src/main/java/org/openRealmOfStars/player/race/SpaceRace.java

```diff
@@ -816,96 +774,6 @@ public int getFoodSpeed(final GravityType gravity) {
     return result;
   }
 
-  /**
-   * Get space race default color
-   * @return PlayerColor
-   */
-  public PlayerColor getPrimaryColor() {
```

Comment:
This color thing is what I am thinking. I am kind of use to that certain space race have certain colors, Of course that can be still do on game start up. But when you are use that certain space race is for example chestnut, in this Reborgian, they match both space ship image, and their leader image.

Then I started to think how other 4X games do this which use colors. CIV series have had certain color associated same civilizations for quite long, but I found thread from Steam and Reddit where the ask how to modify that back to CIV VI. I think this should be brought back at some point when space race is actual class.